.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: python
+python:
+    - "2.7"
+    - "2.6"
+before_install:
+    - git config --global user.email "OpenStack_TravisCI@f5.com"
+    - git config --global user.name "Travis F5 Openstack"
+install:
+    - pip install requests hacking pytest pytest-cov
+script:
+    - flake8 .
+    - py.test --cov --ignore=test/
+notifications:
+    slack:
+      rooms:
+        - f5openstackdev:$SLACK_PROJECT_TOKEN
```

Comment:
Thanks. I added the environment variables (SLACK_PROJECT_TOKEN, SLACK_BUILD_STATUS_TOKEN) and tokens to the Travis settings page (https://travis-ci.org/F5Networks/f5-openstack-lbaasv2-driver/settings).